setup.py

```diff
@@ -15,6 +16,9 @@
     # pypy already includes an implementation of the greenlet module
     install_requires.append('greenlet')
 
+if os.name == 'nt':
```

Comment:
move this before line 10 and change it to `elif sys.version_info < (3, 4):` (trollius is not required if we have pyuv)